src/contracts/middleware/RegistryBase.sol

```diff
@@ -71,7 +71,9 @@ abstract contract RegistryBase is VoteWeigherBase, IQuorumRegistry {
         uint8 _NUMBER_OF_QUORUMS
     ) VoteWeigherBase(_strategyManager, _serviceManager, _NUMBER_OF_QUORUMS)
     // solhint-disable-next-line no-empty-blocks
-    {}
+    {
+        require(_NUMBER_OF_QUORUMS <= 2, "RegistryBase: NUMBER_OF_QUORUMS must be less than or equal to 2");
```

Comment:
if we're bothering with this, probably also want to check it's nonzero

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/7/commits/68f80a0c541b8ba06d37d8840a96fe9e4beee6e1